src/jit/CMakeLists.txt

```diff
@@ -299,7 +301,7 @@ if(WIN32)
   # massage it into the correct format here to create "ClrJit.exports.def".
   set(JIT_EXPORTS_FILE ${CMAKE_CURRENT_BINARY_DIR}/ClrJit.exports.def)
   set(JIT_EXPORTS_FILE_TEMP ${JIT_EXPORTS_FILE}.txt)
-  file(READ "ClrJit.exports" exports_list)
+  file(READ ${CLRJIT_EXPORTS} exports_list)
```

Comment:
I wonder - doesn't the Win32 have the same dependency issue as Linux?

Reply:
We don't support running Ninja on Windows currently for CoreCLR, so that's probably why it wasn't showing up. I'll fix the Windows side as well.

Reply:
Actually I don't think Windows will have the same issue since it generates the final exports file at configure time instead of a build time.

Reply:
Actually, there is an issue on Windows. The way it is now works only when you build using the command line build.cmd that always invokes cmake. However, if you build once using the command line, open the generate CoreCLR.sln in VS and then build from VS, changes in the ClrJit.exports are ignored (I've tried that). If we changed the way we do it on Windows to use the same way as you've changed it for Unix, it would work.
Moreover, it would be nice to have the Windows and Unix ways works in the same way.

Reply:
Sounds good! I'll go fix it.